AppKit/CPTabView.j

```diff
@@ -879,6 +940,48 @@ var CPTabViewItemsKey               = "CPTabViewItemsKey",
     return [super hitTest:aPoint];
 }
 
+// Some CPBox overrides because CPTabView use of CPBox differs from legacy CPBox regarding layout.
+
+- (void)setContentView:(CPView)aView
+{
+    if (aView === _contentView)
+        return;
+
+    if (_contentInset)
+        [aView setFrame:CGRectInsetByInset([self bounds], _contentInset)];
+    [aView setAutoresizingMask:CPViewWidthSizable | CPViewHeightSizable];
+
+    //  A nil contentView is allowed (tested in Cocoa 2013-02-22).
+    if (!aView)
+        [_contentView removeFromSuperview];
+    else if (_contentView)
+        [_boxView replaceSubview:_contentView with:aView];
+    else
+        [_boxView addSubview:aView];
+
+    _contentView = aView;
+
+    [self refreshDisplay];
+}
+
+// FIXME: peut-être besoin de setFrameFromContentFrame
```

Comment:
English please!

Reply:
Oups ! I should have deleted this one... When a comment is in French, well, that's a one for me only... ;-)